NAMESPACE

```diff
@@ -102,6 +102,7 @@ export(importQIIME2)
 export(importQZA)
 export(importTaxpasta)
 export(inner_join)
+export(joint_rpca)
```

Comment:
(exported) function names in the package are in camelCase -> "jointRPCA" would the name here I think. 

-> Update the exported function names